src/Pages/RegistrationPage/RegistrationPage.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import CardWrapper from '../../Atoms/Card/Card';
+import InputField from '../../Atoms/InputField/InputField';
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import RegistrationPageStyled from './RegistrationPageStyled';
+import { useNavigate } from 'react-router-dom';
+
+const RegistrationPage = () => {
+  const [name, setName] = useState('');
+  const [surname, setSurname] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const navigate = useNavigate();
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+
+    const data = { name, surname, email, password };
+
+    //test, after submit, entered user data
+    console.log(data);
+    navigate('/login');
+  };
+
+  return (
+    <CardWrapper>
+      <RegistrationPageStyled onSubmit={submitHandler}>
+        <InputField
+          labelText="Email"
+          htmlFor="email"
+          type="email"
+          placeholder="&#xf0e0; Email"
+          id="email"
+          required
+          value={email}
+          onChange={(event) => setEmail(event.target.value)}
```

Comment:
setStates are repeatible, and there will be more states, more needed useState (look to data -> dummy-data -> competitors.js - there are more infos we need in register form), maybe you can put all of that separately? other function/other hook/custom hook/other way?

Reply:
Use state as i know should be created for every usecase separately. I'll search for the way to do it more nicely :).

Reply:
More state at some point doesn't necessarily improve readability, at some point a consistent one state can reduce the lines of repeatable code. 
Remember you can also use an initial object with all key/value pairs you need, and later overwrite is by setting something similar to:  
`    setFormData({
      ...formData,
      [event.target.name]: event.target.value,
    });`